examples/zkapps/anonymous-message-board/package.json

```
@@ -0,0 +1,49 @@
+{
+  "name": "snarkyjs-tutorial-message",
```

@jackryanservia is this name used in the code? could we rename now? 
```suggestion
  "name": "o1js-tutorial-message",
```